drivers/gl_context/SCsub

```diff
@@ -2,7 +2,8 @@ Import('env')
 Import('env_drivers')
 
 env_drivers.add_source_files(env.drivers_sources,"*.cpp")
-env_drivers.add_source_files(env.drivers_sources,"glew.c")
-
-env_drivers.Append(CFLAGS = ['-DGLEW_STATIC'])
-env_drivers.Append(CPPPATH = ['.'])
+if (env['glew'] == 'yes'):
```

Comment:
Isn't SConscript essentially python? In that case, if `env` is a python `dict`you can use `env.get('glew')` which returns `None` if it's not there and the value if it is.